frontend/src/devicelist/ContentsTable.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Box } from "@material-ui/core"
+
+import * as DeviceApi from "../api/DeviceApi";
+import GenerateData from './SampleData'
+import SearchBar from './SearchBar'
+import SortBy from './SortBy'
+import VerticalTabs from './VerticalTabs'
+import TabPanel from './TabPanel'
+import DevicesTable from './DevicesTable'
+
+export default class ContentsTable extends React.Component {
+    constructor(props) {
+        super(props)
+        var sampleData = GenerateData();
+        this.state = {
+            senders: sampleData.senders,
+            receivers: sampleData.receivers,
+            value: 0
+        }
+        this.handleValueChange = this.handleValueChange.bind(this);
+        this.handleSendersChange = this.handleSendersChange.bind(this);
+        this.handleReceiversChange = this.handleReceiversChange.bind(this);
+    }
+
+    componentDidMount() {
+        DeviceApi.getSenders(this.handleSendersChange);
+        DeviceApi.getReceivers(this.handleReceiversChange);
+    }
+
+    handleValueChange(value) {
+        this.setState({
+            value: value
+        });
+    }
```

Comment:
Can we use a name that is a little more specific here instead of `value`.

Reply:
how about `tabIndex`?